content/applications/inventory_and_mrp/quality/quality_management/quality_control_points.rst

```diff
@@ -1,89 +1,123 @@
-three new fields: :guilabel:`Device`, :guilabel:`Norm`, and :guilabel:`Tolerance`. These fields can
-be configured so that only products within a certain tolerance will pass the check:
-
-- :guilabel:`Device`: select the measuring device that should be used to take the measurement
-  (e.g., measuring tape).
-- :guilabel:`Norm`: specify the desired measurement that the product should conform to and the unit
-  of measurement that should be used
-- :guilabel:`Tolerance`: select the range that a measurement can be within while still passing the
-  check (e.g., :guilabel:`from` 59.5 mm :guilabel:`to` 60.5 mm)
-
-.. image:: quality_control_points/measure_quality_check.png
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+.. |MOs| replace:: :abbr:`MOs (Manufacturing Orders)`
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+.. |QCPs| replace:: :abbr:`QCPs (Quality Control Points)`
+
+In Odoo, quality control points (QCPs), are used to automatically create :doc:`quality checks <quality_checks>`
+at predetermined intervals. |QCPs| can be configured to create quality checks for specific
+operations (manufacturing, delivery, etc.), as well as specific products within those operations.
+
+Using |QCPs| allows quality teams to ensure that products are being regularly inspected for defects
+and other issues.
+
+Configure a quality control point
+=================================
+
+To create a new |QCP|, navigate to :menuselection:`Quality --> Quality Control --> Control Points`,
+and then click :guilabel:`New`.
+
+Begin filling out the new |QCP| by entering a unique :guilabel:`Title` that will make the |QCP|
+easily identifiable.
+
+In the :guilabel:`Products` field, select one or more products that the |QCP| should apply to. If
+the |QCP| should apply to an entire product category, select it in the :guilabel:`Product
+Categories` field.
+
+In the :guilabel:`Operations` field, select the operation(s) that will trigger the |QCP|. For
+example, selecting the *Manufacturing* operation will cause a quality check to be created for new
+|MOs|.
+
+.. note::
+   When creating a new |QCP|, at least one operation must be listed in the :guilabel:`Operations`
+   field. However, the :guilabel:`Products` and :guilabel:`Product Categories` fields can be left
+   blank. If they are, the |QCP| will generate quality checks for every instance of the specified
+   operation(s).
+
+If the *Manufacturing* operation is selected in the :guilabel:`Operations` field, a new field
+appears below it, titled :guilabel:`Work Order Operation`. Select a specific work order to generate
+quality checks for that operation, rather than the manufacturing operation in general.
+
+For example, a |QCP| could be configured to create quality checks for the *Assembly* work order of
+the *Coffee Table* product. When a new |MO| is confirmed for a *Coffee Table*, the |QCP| creates a
+quality check for the *Assembly* operation specifically.
```

Comment:
Very minor thing, but user inputs like the name of a product or an operation that's not available in every Odoo database should be wrapped in backticks `

```
So *Coffee Table* --> `Coffee Table`
```
and I'd change every instance of these and \*Assembly\* in the doc!

According to this [writers workshop](https://docs.google.com/document/d/1laiWC9ow0IjXPuQp5h8L_Tj98HbE8ymGn9v-FFOwlYM/edit#heading=h.ipar7km38ci3)